eq-author/src/components/ContentPickerv2/Menu.js

```diff
@@ -56,35 +56,38 @@ export const ParentMenuItem = styled(MenuItem)`
     right: 0.5em;
     opacity: 0.5;
   }
-
-  &[aria-selected="true"] {
-    background: #f3f3f3;
-    &::after {
-      opacity: 1;
-    }
-    &:hover {
-      background: #f3f3f3;
-    }
-  }
+  ${props =>
+    props["aria-selected"] &&
+    css`
+      background-color: #f3f3f3;
+      &::after {
+        opacity: 1;
+      }
+      &:hover {
+        background: #f3f3f3;
+      }
+    `}
 `;
 
 export const SubMenuItem = styled(MenuItem)`
   --color: ${colors.text};
   --colorSecondary: ${colors.textLight};
   --colorTertiary: ${colors.text};
 
-  &[aria-selected="true"] {
-    --color: ${colors.white};
-    --colorSecondary: ${colors.white};
+  ${props =>
+    props["aria-selected"] &&
+    css`
+      background-color: ${colors.primary};
+      --color: ${colors.white};
+      --colorSecondary: ${colors.white};
 
-    background: ${colors.primary};
-    border-color: #377090;
-    z-index: 2;
+      border-color: #377090;
```

Comment:
Le same

Reply:
Let's not hard code them innit

Reply:
We currently don't have that in the theme but it's a blue so I guess. ${colors.blue} would do it